partitioner/src/main/java/io/confluent/connect/storage/partitioner/PartitionerConfig.java

```diff
@@ -78,10 +82,20 @@
   public static final String TIMEZONE_DEFAULT = "";
   public static final String TIMEZONE_DISPLAY = "Timezone";
 
-  // Schema group
-  public static final String SCHEMA_GENERATOR_CLASS_CONFIG = "schema.generator.class";
-  public static final String SCHEMA_GENERATOR_CLASS_DOC = "The schema generator to use with partitioners.";
-  public static final String SCHEMA_GENERATOR_CLASS_DISPLAY = "Schema Generator Class";
+  public static final String TIMESTAMP_EXTRACTOR_CLASS_CONFIG = "timestamp.extractor";
+  public static final String TIMESTAMP_EXTRACTOR_CLASS_DOC =
+      "The extractor that gets the timestamp for records when partitioning with ``TimeBasedPartitioner``. It can be " +
+          "set to ``Wallclock``, ``Record`` or ``RecordField`` in order to use one of the built-in timestamp " +
+          "extractors or be given the fully-qualified class name of a user-defined class that extends the" +
+          "``TimestampExtractor`` interface.";
+  public static final String TIMESTAMP_EXTRACTOR_CLASS_DEFAULT = "Wallclock";
```

Comment:
No, the changes here need to happen in lock-step with the changes in the equivalent PR in storage-cloud: https://github.com/confluentinc/kafka-connect-storage-cloud/pull/19

The section in the link you mention above, becomes:
https://github.com/kkonstantine/kafka-connect-storage-cloud/blob/29f3da40ebea2ef4f1eb3a0f733c9867e7f1148f/kafka-connect-s3/src/main/java/io/confluent/connect/s3/TopicPartitionWriter.java#L164